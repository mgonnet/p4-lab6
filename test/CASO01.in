
ROOT
123

aru
4855460
juan
perez
m
21
02
1993
ADMIN
0
cs
SALIR

