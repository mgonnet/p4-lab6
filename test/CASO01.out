[H[2JBIENVENIDO A HOSPITAL PÃ‰CUATRO
------------------------------
Presione enter para iniciar sesion o escriba SALIR para salir...
[H[2JINCIO DE SESION
---------------
Ingrese su CI:
> [H[2JINCIO DE SESION
---------------
Eres el administrativo por defecto.
Por favor ingresa tu contraseÃ±a
> [H[2JINCIO DE SESION
---------------
Tu sesion ha sido iniciada. Enter para continuar
[H[2JSELECCIONE ACCION
-----------------
(aru) > Alta/ReactivaciÃ³n Usuario
(uar) > Usuarios dados de alta y reactivados
(am)  > Alta Medicamento
(ard) > Alta RepresentaciÃ³n estandarizada de DiÃ¡gnosticos
(rc)  > Registro Consulta
(ccd) > Cantidad de Consultas por CategorÃ­a DiagnÃ³stico
(lre) > Listar Representaciones Estandarizadas
(cs) > Cerrar Sesion
> [H[2JALTA/REACTIVACION DE USUARIOS
-----------------------------
EstÃ¡ aquÃ­ para dar de alta o reactivar un usuario.
Ingrese la cedula de usuario en cuestion:
> El nombre es: JUAN
El apellido es: MONTES
El sexo es: MASCULINO
La edad es: 0
El usuario estÃ¡ inactivo

[H[2JALTA/REACTIVACION DE USUARIOS
-----------------------------
Este usuario estÃ¡ inactivo. Â¿Desea reactivarlo? [1/0]
> [H[2JALTA/REACTIVACION DE USUARIOS
-----------------------------
Â¿Desea probar suerte con otro usuario? [1/0]
> [H[2JSELECCIONE ACCION
-----------------
(aru) > Alta/ReactivaciÃ³n Usuario
(uar) > Usuarios dados de alta y reactivados
(am)  > Alta Medicamento
(ard) > Alta RepresentaciÃ³n estandarizada de DiÃ¡gnosticos
(rc)  > Registro Consulta
(ccd) > Cantidad de Consultas por CategorÃ­a DiagnÃ³stico
(lre) > Listar Representaciones Estandarizadas
(cs) > Cerrar Sesion
> [H[2JSELECCIONE ACCION
-----------------
(aru) > Alta/ReactivaciÃ³n Usuario
(uar) > Usuarios dados de alta y reactivados
(am)  > Alta Medicamento
(ard) > Alta RepresentaciÃ³n estandarizada de DiÃ¡gnosticos
(rc)  > Registro Consulta
(ccd) > Cantidad de Consultas por CategorÃ­a DiagnÃ³stico
(lre) > Listar Representaciones Estandarizadas
(cs) > Cerrar Sesion
> [H[2JSELECCIONE ACCION
-----------------
(aru) > Alta/ReactivaciÃ³n Usuario
(uar) > Usuarios dados de alta y reactivados
(am)  > Alta Medicamento
(ard) > Alta RepresentaciÃ³n estandarizada de DiÃ¡gnosticos
(rc)  > Registro Consulta
(ccd) > Cantidad de Consultas por CategorÃ­a DiagnÃ³stico
(lre) > Listar Representaciones Estandarizadas
(cs) > Cerrar Sesion
> [H[2JSELECCIONE ACCION
-----------------
(aru) > Alta/ReactivaciÃ³n Usuario
(uar) > Usuarios dados de alta y reactivados
(am)  > Alta Medicamento
(ard) > Alta RepresentaciÃ³n estandarizada de DiÃ¡gnosticos
(rc)  > Registro Consulta
(ccd) > Cantidad de Consultas por CategorÃ­a DiagnÃ³stico
(lre) > Listar Representaciones Estandarizadas
(cs) > Cerrar Sesion
> [H[2JSELECCIONE ACCION
-----------------
(aru) > Alta/ReactivaciÃ³n Usuario
(uar) > Usuarios dados de alta y reactivados
(am)  > Alta Medicamento
(ard) > Alta RepresentaciÃ³n estandarizada de DiÃ¡gnosticos
(rc)  > Registro Consulta
(ccd) > Cantidad de Consultas por CategorÃ­a DiagnÃ³stico
(lre) > Listar Representaciones Estandarizadas
(cs) > Cerrar Sesion
> [H[2JSELECCIONE ACCION
-----------------
(aru) > Alta/ReactivaciÃ³n Usuario
(uar) > Usuarios dados de alta y reactivados
(am)  > Alta Medicamento
(ard) > Alta RepresentaciÃ³n estandarizada de DiÃ¡gnosticos
(rc)  > Registro Consulta
(ccd) > Cantidad de Consultas por CategorÃ­a DiagnÃ³stico
(lre) > Listar Representaciones Estandarizadas
(cs) > Cerrar Sesion
> [H[2JSELECCIONE ACCION
-----------------
(aru) > Alta/ReactivaciÃ³n Usuario
(uar) > Usuarios dados de alta y reactivados
(am)  > Alta Medicamento
(ard) > Alta RepresentaciÃ³n estandarizada de DiÃ¡gnosticos
(rc)  > Registro Consulta
(ccd) > Cantidad de Consultas por CategorÃ­a DiagnÃ³stico
(lre) > Listar Representaciones Estandarizadas
(cs) > Cerrar Sesion
> [H[2JSELECCIONE ACCION
-----------------
(aru) > Alta/ReactivaciÃ³n Usuario
(uar) > Usuarios dados de alta y reactivados
(am)  > Alta Medicamento
(ard) > Alta RepresentaciÃ³n estandarizada de DiÃ¡gnosticos
(rc)  > Registro Consulta
(ccd) > Cantidad de Consultas por CategorÃ­a DiagnÃ³stico
(lre) > Listar Representaciones Estandarizadas
(cs) > Cerrar Sesion
> [H[2JBIENVENIDO A HOSPITAL PÃ‰CUATRO
------------------------------
Presione enter para iniciar sesion o escriba SALIR para salir...
